### PR TITLE
feat(deps): Add Nextcloud 31 support

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

**Review by kesselb — APPROVED** [review]

**Comment by kesselb** [comment]
Lowered min-version to 30

File: appinfo/info.xml [hunk 1]
```diff
@@ -34,7 +34,7 @@ The rating depends on the installed text processing backend. See [the rating ove
 
 Learn more about the Nextcloud Ethical AI Rating [in our blog](https://nextcloud.com/blog/nextcloud-ethical-ai-rating/).
 	]]></description>
-	<version>4.0.0-alpha.3</version>
+	<version>5.0.0-dev.0</version>
```
Comment (kesselb) [inline appinfo/info.xml:37]: Check with Christoph if we want to go with 5.0.0 or 4.1.0

**Review by kesselb — CHANGES_REQUESTED** [review]

File: appinfo/info.xml (hunk 1 above)
Comment (ChristophWurst) [reply appinfo/info.xml:37]: 4.1 as long as we keep 30 as min

**Review by kesselb — APPROVED** [review]

File: appinfo/info.xml [hunk 2]
```diff
@@ -34,7 +34,7 @@ The rating depends on the installed text processing backend. See [the rating ove
 
 Learn more about the Nextcloud Ethical AI Rating [in our blog](https://nextcloud.com/blog/nextcloud-ethical-ai-rating/).
 	]]></description>
-	<version>4.0.0-beta.2</version>
+	<version>4.1.0-dev.0</version>
```
Comment (ChristophWurst) [inline appinfo/info.xml:37]: can we use alpha? I think the versions are sorted alphabetically. If we start with dev we can only go to RC and not to alpha or beta

**Pull request merged**